### ISSUE TITLE
Empty space under the Header

### ISSUE THREAD

**Issue by Mr7Cat** [react-native-community/react-native-tab-view#316]
"react-native": "0.41.2"
react-native-tab-view@0.0.67

When nested ListView in the tab, empty space appear under the header when show the page first time.
If scroll list view quickly, it happen too.

It seems
`bounces={false}
          bouncesZoom={false}
          alwaysBounceVertical={false}`
not work?

### Code sample, screenshots
<img width="375" alt="wx20170727-160815 2x" src="https://user-images.githubusercontent.com/1405850/28660691-98f6f4a2-72e6-11e7-8475-33c971486014.png">

```
 <TabViewAnimated
                    style={styles.tabView}
                    navigationState={this.state}
                    renderScene={this._renderScene}
                    renderHeader={this._renderHeader}
                    onRequestChangeTab={this._handleChangeTab}
                >
........

_renderScene = ({ route }) => {
        switch (route.key) {
            case '1':
                return (
                    <ChatRoomListContainer
                        style={styles.container}
                        {...this.props}
                        showAlertView={this.showAlertView}
                        gotoRosterList={this.gotoRosterList} />
                );
            case '2':
                return (
                    <RosterListContainer style={styles.container} {...this.props} />
                );
            default:
                return null;
        }
    }

.......

 <ListView
          automaticallyAdjustContentInsets={false}
          bounces={false}
          bouncesZoom={false}
          alwaysBounceVertical={false}
          enableEmptySections={true}
          initialListSize={this.props.default_init_list_size}
          dataSource={this.state.dataSource.cloneWithRows(rooms)}
          renderRow={this._renderItem}
          style={styles.list_view}
          removeClippedSubviews={Platform.select({
            ios: false,
            android: true
          })}
        ></ListView>
```

**Comment by gang544043963** [comment]
I met the same question！Empty space appeared only if the scene is listView.

**Comment by ixje** [comment]
Try having a look at [Example 5.](https://github.com/react-native-community/react-native-tab-view/blob/master/example/src/ScrollViewsExample.js). It uses ListViews and does not show your behaviour. Don't be mislead by the title of the source file as in the `renderScene` function it returns a [BasicListView](https://github.com/react-native-community/react-native-tab-view/blob/master/example/src/BasicListView.js) which as the name suggests is a ListView.

You're react-native version also seems pretty old, so perhaps you can try upgrading that to see if there have been any fixes related to it.

While this might not be a direct answer, alternatively you try if the same happens with a FlatList. ListView's will be [depreciated in the future and replaced by better alternatives such as FlatList](https://facebook.github.io/react-native/blog/2017/03/13/better-list-views.html).

> No more ListViews or DataSources, stale rows, ignored bugs, or excessive memory consumption - with the latest React Native March 2017 release candidate (0.43-rc.1) you can pick from the new suite of components what best fits your use-case, with great perf and feature sets out of the box

**Comment by satya164** [comment]
Hey, I just released a new alpha `2.0.0-alpha.0` of the library. It's rewritten using [react-native-gesture-handler](https://github.com/kmagiera/react-native-gesture-handler) and [react-native-reanimated](https://github.com/kmagiera/react-native-reanimated) addresses a many platform specific bugs and performance problems.

Please try the new version and see if it addresses your issue. If not, please open a new issue following the issue template.
